samples/XCT.Sample/ViewModels/SettingViewModel.cs

```diff
@@ -44,7 +47,7 @@ void LoadLanguages()
 				{ new Language(AppResources.English, "en") },
 				{ new Language(AppResources.Spanish, "es") }
 			};
-			SelectedLanguage = SupportedLanguages.FirstOrDefault(pro => pro.CI == LocalizationResourceManager.Current.CurrentCulture.TwoLetterISOLanguageName);
+			SelectedLanguage = SupportedLanguages.Single(pro => pro.CI == LocalizationResourceManager.Current.CurrentCulture.TwoLetterISOLanguageName);
```

Comment:
This will crash if the app is using a different language. I, for example, am using "pt-BR" as Language, and that Single can't return a match result, that way the app crashes on me. Can you add a verification?
This was my workaround, isn't pretty but can give you some inspiration. 

```csharp
			Language? s = SupportedLanguages.FirstOrDefault(pro => pro.CI == LocalizationResourceManager.Current.CurrentCulture.TwoLetterISOLanguageName);

			if (s == null)
			{
				LocalizationResourceManager.Current.SetCulture(new CultureInfo("en"));
				s = new Language("Enlighs", "en");
			}

			SelectedLanguage = s.Value;
```

Reply:
Missed that.. I think, I'll just revert that change since it's out of scope of this PR, and fix just brings unneeded complexity